docs/02-interface-values/05-underlying-type.md

```diff
@@ -109,7 +109,7 @@ Huh, it looks like there is some duplicate information. Notice how the addresses
 * **`0x459d80`**: type address for `int32(3)` and `int32(5)`
 * **`0x459dc0`**: type address for `int64(3)` and `int64(5)`
 
-This is because type information is global and is shared for all values stored in interfaces. However, are...are the addresses for the values _also_ duplicated -- across _type_!?
+This is because type information is global and is shared for all values stored in interfaces. However, are the addresses for the values _also_ shared -- across _type_!?
```

Comment:
I got confused by `duplicated` since the value IMHO is actually `shared` and you use the same terminology in the `type` section. Hence I changed it from `duplicated`->`shared`